policy/policy_handler/handler.go

```diff
@@ -113,8 +115,16 @@ func (h EventHandler) evaluate(ctx context.Context, req skill.RequestContext, da
 		paramValues[p.Name] = p.Value
 	}
 
+	// atm-skill local appends the current user's name to the skill name
+	// we can strip that suffix off before calling evalSelector to let it match on the original name
+	goalDefName := goalName
+	u, err := user.Current()
+	if err == nil && strings.HasSuffix(goalDefName, fmt.Sprintf("-%s", u.Username)) {
```

Comment:
Could we use `TrimSuffix` instead?

Reply:
Yep! 22fc6f86ce211c4ef473c6dff24b629723eff15b